src/jira/boards.js

```diff
@@ -0,0 +1,20 @@
+const config = require("../config");
+const notifyConfig  = config.get("notify");
+var JiraClient = require('jira-connector');
+
+var jira = new JiraClient( {
+    host: notifyConfig.jira.host,
+    basic_auth: {
+        base64: notifyConfig.jira.basic_auth.base64
+    }
+});
+
+jira.board.getAllBoards({}, function callback(no, data, response) {
```

Comment:
What's `no` as a value? Could this have a clearer name?

Reply:
The `no` variable was recommended by the NPM Package; but you are correct, it's not terribly descriptive. This variable is **always** `null`; so I've renamed it to `empty`.